buildSrc/src/main/java/Config.kt

```diff
@@ -2,16 +2,16 @@
 import java.math.BigDecimal
 
 object Config {
-    val AGP = System.getenv("VERSION_AGP") ?: "7.4.2"
-    val kotlinVersion = "1.8.0"
+    val AGP = System.getenv("VERSION_AGP") ?: "8.6.0"
+    val kotlinVersion = "1.9.22"
     val kotlinStdLib = "stdlib-jdk8"
 
     val springBootVersion = "2.7.5"
     val springBoot3Version = "3.3.2"
-    val kotlinCompatibleLanguageVersion = "1.4"
+    val kotlinCompatibleLanguageVersion = "1.5"
```

Comment:
I guess this deserves a "Breaking changes" note at least? doubt anyone is still on this version, but gotta call it out (or make it part of v8?)